Alicerce/Sources/Logging/Log.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  Alicerce.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 03/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public final class Log {
+
+    private static var providers = [LogProvider]()
```

Comment:
We can make this a `private(set) var providers`, so it can be read from the outside (including the element count)